### PR TITLE
Airbrake simulator

### PR DESCRIPTION
**Pull request by mstoelzle**
Fully implemented the airbrake simulator now:
1. `airbrake_state` (part of the `rocket_state`): `[ab_position, ab_velocity, ab_motor_current]`
2. The airbrake simulator computes the derivate of the `airbrake_state`. This derivative is integrated by the scipy `solve_ivp` (rutta-kutta)
3. The limits of the airbrake are not modelled by a spring-damper element, but rather the following measures are implemented:
a) The position of the airbrake in the rocket state is set to be within the allowed range [0, 1] at all times
b) If the airbrake position is at the bounds or out-of-bounds and at the same time the velocity or acceleration tries to further push the airbrake position out-of-bounds, the velocity and acceleration are set to zero

### CODE REVIEW

File: src/eulersimcon_backend/components/simulation/airbrake_simulator.py [hunk 1]
```diff
@@ -13,15 +13,42 @@ def run(self, rocket_state: RocketState, control_step: ControlStep = None) -> np
...
+        C1 = 1 / ((217 ** 2) * 0.75 * 14 * 98)  # 217=average velocity for 0.7 Mach, 0.75=gear efficiency, 14=gear ratio
+        C2 = 4.2  # friction force at Ma 0.7
+        C3 = 11.3  # pressure force at Ma 0.7
+
+        # Kirchhoff equation
+        current_dot = 1 / L * (C * control_input - k * rocket_state.airbrake_state[1] - R *
+                               rocket_state.airbrake_state[2])
+        # Euler equation
+        ab_velocity_dot = 1 / J * (
+                    rocket_state.airbrake_state[0] * V / C1 * (C2 * np.sign(rocket_state.airbrake_state[1]) + C3) - d *
```
Comment (TunVunDerBunn) [inline src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: should rocket_state.airbrake_state[0] not be replaced with control_input?

File: src/eulersimcon_backend/components/simulation/airbrake_simulator.py [hunk 2]
```diff
@@ -34,15 +33,15 @@ def run(self, rocket_state: RocketState, control_step: ControlStep = None) -> np
         #  Inertia of entire AB System 61g=mass per gear, 323=mass per scoop+rail
         J = 13.8E-7 + 0.8E-7 + (0.01 ** 2) / 1000 * (61 + 4 * 323)
         d = 0.01  # No value in Maxon catalogue--> guess, will try to get more accurate value soon
-        C1 = 1 / ((217 ** 2) * 0.75 * 14 * 98)  # 217=average velocity for 0.7 Mach, 0.75=gear efficiency, 14=gear ratio
+        C1 = (217 ** 2) * 0.75 * 14  # 217=average velocity for 0.7 Mach, 0.75=gear efficiency, 14=gear ratio
```
Comment (TunVunDerBunn) [inline src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: is it normal that we inverse C1 here and we do not change anything in line 45

File: src/eulersimcon_backend/components/simulation/airbrake_simulator.py (hunk 1 above)
Comment (mstoelzle) [reply src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: @christophdomenig98 can you respond to that?

File: src/eulersimcon_backend/components/simulation/airbrake_simulator.py (hunk 2 above)
Comment (mstoelzle) [reply src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: @TunVunDerBunn You are commenting on an outdated file. You have to refresh the pull request

File: src/eulersimcon_backend/components/simulation/airbrake_simulator.py (hunk 1 above)
Comment (christophdomenig98) [reply src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: the V term needs to be rocket_state.V_x**2 + rocket_state.V_y**2+ rocket_state.V_z**2
but this is not the newest version

Comment (christophdomenig98) [reply src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: I don't know how git works, but I gave Maxi a newer version with a lot of changes yesterday evening

Comment (TunVunDerBunn) [reply src/eulersimcon_backend/components/simulation/airbrake_simulator.py]: yes accidentally commented on an old version I am sorry

**Pull request merged**